shell/platform/windows/flutter_windows_view.h

```diff
@@ -27,6 +28,9 @@
 
 namespace flutter {
 
+// ID for the window frame buffer.
+static constexpr uint32_t kWindowFrameBufferID = 0;
```

Comment:
Nothing in a header should be declared static; this is an invitation to ODR violations. See https://abseil.io/tips/140

You want `inline constexpr`, not `static constexpr`